utils/generic.py

```diff
@@ -232,6 +232,46 @@ def filter_low_conf_regions(mt: Union[hl.MatrixTable, hl.Table], filter_lcr: boo
     return mt
 
 
+def vep_or_lookup_vep(ht, reference_vep_ht=None, reference=None, vep_config=None):
```

Comment:
Should you pass through `block_size`, `name` and `csq`? https://hail.is/docs/0.2/methods/genetics.html?highlight=vep#hail.methods.vep

Reply:
Given that we've never used block_size or name, and I expressly do not want to allow `csq` until we think through how we could have a second VEP file with the string schema, I'm tempted not to for now.